### PR TITLE
Refactor header bar

### PR DESCRIPTION
**Pull request by danni**

### CODE REVIEW

File: src/pages/HomePage.js [hunk 1]
```diff
@@ -8,16 +7,34 @@ class HomePage extends React.Component {
 
     render(): React.Element {
         return (
-            <div className="Page">
-                <div className="Page-header">
-                    <HeaderBar />
+            <div className="HomePage">
+                <div className="header">
+                    <div className="branding-container">
```
Comment (DanielHeath) [inline src/pages/HomePage.js:12]: Should the brand/logo stuff still be a component?

Comment (danni) [inline src/pages/HomePage.js:12]: It only appears once, it may as well be on the page IMO.

File: src/pages/CategoryPage.js [hunk 2]
```diff
@@ -60,7 +60,6 @@ class CategoryPage extends React.Component {
     render(): React.Element {
         return (
             <div>
-                <Router.RouteHandler />
```
Comment (DanielHeath) [inline src/pages/CategoryPage.js:63]: Cool - just bear in mind if we nest anything under this URL it won't render.

File: src/pages/HomePage.js (hunk 1 above)
Comment (DanielHeath) [inline src/pages/HomePage.js:12]: Fair

**Comment by DanielHeath** [comment]
++

File: src/pages/CategoryPage.js (hunk 2 above)
Comment (danni) [inline src/pages/CategoryPage.js:63]: It was really inconsistently used. It seemed better to clean it up for now.

**Pull request merged**